app/instrumentation-tests/src/org/commcare/androidTests/DateWidgetsTests.kt

```diff
@@ -137,7 +137,7 @@ class DateWidgetsTests: BaseTest() {
      * function returns different format of the selected date
      */
     fun getDateInDifferentFormats(date : Date) : Array<String>{
-        val longFormat = SimpleDateFormat("EE, MMM dd, yyyy").format(date)
+        val longFormat = SimpleDateFormat("EE, MMM d, yyyy").format(date)
```

Comment:
does this now work when the date is in double digits ?

Reply:
Yes, it does. Its now similar to the format used in shortFormat, it works fine for both single and double digit dates.